spec/dummy/app/views/layouts/dummy_admin_layout.html.erb

```diff
@@ -18,5 +18,6 @@
       <%= yield %>
     </main>
   </div>
+  <%= render "govuk_publishing_components/components/feedback" %>
```

Comment:
I'm not sure we want this in the `dummy_admin_layout.html.erb` (as we won't have it in an admin app), maybe the dummy (public-facing) `application.html.erb` layout would be more suitable.

Reply:
Okay, thanks, have changed.